pilot/pkg/networking/networking.go

```diff
@@ -102,6 +110,8 @@ type FilterChain struct {
 	HTTP []*http_conn.HttpFilter
 	// Thrift is the set of Thrift filters for this filter chain
 	Thrift []*thrift_proxy.ThriftFilter
+	// Dubbo is the set of Dubbo filters for this filter chain
+	Dubbo []*dubbo_proxy.DubboFilter
```

Comment:
also remove

Reply:
Unlike mysql/mangodb, Dubbo filter handles the routing by itself instead of passing it through to a tcp proxy filter, so the configuration generation can't be pused up to buildInboundNetworkFilters.